src/components/config/EditChipList.vue

```diff
@@ -6,48 +6,50 @@
 
 <template>
   <v-card variant="elevated" class="text-left" flat>
-    <v-card-title class="d-flex">
-      <div>{{ propsVal.title }}</div>
-      <template v-if="propsVal.needTip && propsVal.tipContent"
-        ><TipPopover type="M" :tip-content="propsVal.tipContent"></TipPopover
-      ></template>
-    </v-card-title>
+    <v-card-item class="mb-10">
+      <v-card-title class="d-flex">
+        <div>{{ propsVal.title }}</div>
+        <template v-if="propsVal.needTip && propsVal.tipContent"
+          ><TipPopover type="M" :tip-content="propsVal.tipContent"></TipPopover
+        ></template>
+      </v-card-title>
+    </v-card-item>
 
-    <v-card-text>
-      <template v-for="(val, idx) in initValArr.valArr || []" :key="idx">
-        <v-hover v-slot="{ isHovering, props }">
-          <v-chip
-            v-bind="props"
-            :value="val"
-            class="ma-2"
-            :color="propsVal.defaultItems?.includes(val) ? 'default' : 'blue'"
-            ><template v-if="isHovering && !propsVal.defaultItems?.includes(val)" #append>
-              <v-icon
-                class="ml-2 display-keywords"
-                color="blue"
-                icon="mdi-close-circle"
-                size="small"
-                variant="plain"
-                @click="deleteItem(idx)"></v-icon>
-            </template>
-            {{ val }}</v-chip
-          ></v-hover
-        >
-      </template>
-      <span v-if="propsVal.needAdd" class="ma-2">
-        <v-btn
-          v-if="!isAdding"
+    <v-card-text v-if="initValArr.valArr.length > 0" class="mt-n8">
+      <v-chip-group v-model="chipGroupModelVal" column multiple class="ma-2">
+        <v-chip
+          v-for="(val, idx) in initValArr.valArr || []"
+          :key="idx"
+          :value="val"
+          :disabled="propsVal.defaultItems?.includes(val)"
           color="blue"
-          icon="mdi-plus"
-          class="add-keywords"
-          variant="text"
-          @click="isAdding = true"></v-btn>
-      </span>
+          class="mb-4"
+          filter
+          :variant="propsVal.defaultItems?.includes(val) ? `tonal` : `outlined`"
```

Comment:
The default black outline for a non-default keyword is kind of discord.
![image](https://github.com/lingyuncai/webinizer-webclient/assets/79566269/4c943e5c-b373-4cc7-8016-e89a64a35271)

Can we still consider below styles in default mode?
![image](https://github.com/lingyuncai/webinizer-webclient/assets/79566269/c05b9341-582a-4af2-816e-55d46eb13914)